tests/utils/exec/rosacli/version_service.go

```diff
@@ -122,14 +126,15 @@ func (v *versionService) ListAndReflectVersions(
 	return versionList, err
 }
 func (v *versionService) ListAndReflectJsonVersions(
-	channelGroup string, hostedCP bool, flags ...string) (versionList *OpenShiftVersionTableList, err error) {
+	channelGroup string, hostedCP bool, flags ...string) (versionList []*OpenShiftVersionJsonOutput, err error) {
 	var output bytes.Buffer
+	flags = append(flags, "-ojson")
```

Comment:
An empty space between '-o' and 'json'?

Reply:
@yuwang-RH It works well for "-ojson". Is it required to add space?

Reply:
If it works as well, that's fine for me